cocos/rendering/instanced-buffer.ts

```diff
@@ -86,6 +101,11 @@ export class InstancedBuffer {
             shader = subModel.shaders[passIdx];
         }
         const descriptorSet = subModel.descriptorSet;
+        const hash = (0 << 30) | (subModel.passes[passIdx].priority as number) << 16 | (subModel.priority as number) << 8 | passIdx;
```

Comment:
What's the purpose of (0 << 30) ? 
0 << 30 is still 0, and it doesn't affect following `|`  bit operations.